### PR TITLE
fix(route): JAVLibrary

### LINKED ISSUE #8936: javlibrary貌似失效了?
### 路由地址

rsshub/javlibrary

### 完整路由地址

https://rsshub.app/javlibrary/stars/aegeq

### 相关文档

https://rsshub.app/javlibrary/stars/aegeq

### 预期是什么？

无法抓取内容

### 实际发生了什么？

Looks like something went wrong

### 部署

RSSHub 演示 (https://rsshub.app)

### 部署相关信息

_No response_

### 额外信息

```shell
Route requested: /javlibrary/stars/aegeq
Error message: Response code 403 (Forbidden): target website might be blocking our access, you can host your own RSSHub instance for a better usability.
Helpful Information to provide when opening issue: 
Path: /javlibrary/stars/aegeq
Node version: v16.3.0
        
如果您认为 RSSHub 导致了该错误，请在Github按照模版，复制本页面信息进行汇报
If you believe this is an error caused by RSSHub, please report on github
        
在线文档与支持，请访问docs.rsshub.app。

For online documentation and support please refer to docs.rsshub.app.
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #8936

## 完整路由地址 / Example for the proposed route(s)

```routes
/javlibrary/update/cn
/javlibrary/newrelease/cn
/javlibrary/newentries/cn
/javlibrary/mostwanted/cn
/javlibrary/bestrated/cn
/javlibrary/genre/amjq/cn
/javlibrary/star/abbds/cn
/javlibrary/bestreviews/cn
/javlibrary/userposts/mangudai/cn
/javlibrary/userwanted/mangudai/cn
/javlibrary/userwatched/mangudai/cn
/javlibrary/userowned/mangudai/cn
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [x] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by TonyRL** [comment]
Can you make it backwards compatible with current routes?
```
/javlibrary/videos/:vtype
/javlibrary/stars/:sid
/javlibrary/users/:uid/:utype
/javlibrary/bestreviews
```

**Comment by nczitzk** [comment]
> Can you make it backwards compatible with current routes?
> 
> ```
> /javlibrary/videos/:vtype
> /javlibrary/stars/:sid
> /javlibrary/users/:uid/:utype
> /javlibrary/bestreviews
> ```

Already done 😃

File: lib/v2/javlibrary/router.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    router.get('/update/:language?', require('./update'));
+    router.get('/newrelease/:language?/:mode?', require('./newrelease'));
+    router.get('/newentries/:language?', require('./newentries'));
+    router.get('/mostwanted/:language?/:mode?', require('./mostwanted'));
+    router.get('/bestrated/:language?/:mode?', require('./bestrated'));
+    router.get('/genre/:genre?/:language?/:mode?', require('./genre'));
+    router.get('/star/:id/:language?/:mode?', require('./star'));
+    router.get('/:type/:id/:language?', require('./user'));
+};
```
Comment (TonyRL) [inline lib/v2/javlibrary/router.js:20]: Please sort the routes in alphabetical order for both old routes and new routes in two sections.

File: lib/v2/javlibrary/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    '/update/:language?': ['nczitzk'],
+    '/newrelease/:language?/:mode?': ['nczitzk'],
+    '/newentries/:language?': ['nczitzk'],
+    '/mostwanted/:language?/:mode?': ['nczitzk'],
+    '/bestrated/:language?/:mode?': ['nczitzk'],
+    '/bestreviews/:language?/:mode?': ['nczitzk'],
+    '/genre/:genre?/:language?/:mode?': ['nczitzk'],
+    '/star/:id/:language?/:mode?': ['nczitzk'],
+    '/:type/:id/:language?': ['nczitzk'],
+};
```
Comment (TonyRL) [inline lib/v2/javlibrary/maintainer.js:11]: Please sort the routes in alphabetical order.

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Pull request merged**